client.go

```diff
@@ -29,14 +28,325 @@ func main() {
 	config.Cluster.Cloud.Token = "YOUR_CLUSTER_DISCOVERY_TOKEN"
 	config.Stats.Enabled = true
 	config.Stats.Period = types.Duration(time.Second)
-
 	client, err := hazelcast.StartNewClientWithConfig(ctx, config)
 	if err != nil {
 		panic(err)
 	}
 	defer client.Shutdown(ctx)
-	log.Println("Connection Successful!")
-	log.Println("Now, `map` will be filled with random entries.")
+	fmt.Println("Connection Successful!")
+
+	// Uncomment the example you want to run.
+	mapExample(client)
+	//sqlExample(client)
+	//jsonSerializationExample(client)
+	//nonStopMapExample(client)
+}
+
+// mapExample shows how to work with Hazelcast maps.
+// It simply puts a bunch of entry to map.
+func mapExample(client *hazelcast.Client) {
+	ctx := context.Background()
+	cities, err := client.GetMap(ctx, "cities")
+	if err != nil {
+		panic(err)
+	}
+	cities.Put(ctx, "1", model.CityAsJson("United Kingdom", "London", 9_540_576))
```

Comment:
I know you've translated the Java example, but how about using a simple function to create the JSON, instead of the `model` package defined in this PR, something like:
```
func city(country, name string, population int64) serialization.JSON {
   // ...
}
```

Reply:
In this way we can eliminate type definitions for City and Country, right? It can simply create a JSON formatted string and then convert it to serialization.JSON object.

Reply:
Right, it's better to write the smallest code we can write. Especially in this case, it will be easier for the user to understand what's going on.